freqtrade/tests/conftest.py

```diff
@@ -39,10 +39,24 @@ def log_has_re(line, logs):
                   False)
 
 
-def get_args(args) -> List[str]:
+def get_args(args):
     return Arguments(args, '').get_parsed_arg()
 
 
+def patched_configuration_open(mocker, config):
+    file_mock = mocker.patch('freqtrade.configuration.configuration.open', mocker.mock_open(
```

Comment:
Ideally we should only have one of these 2 helper functions (and prefferably the other one)...

Reply:
Done, see also comment above.